lib/que/migrations.rb

```diff
@@ -48,14 +48,41 @@ def db_version
           # No table in the database at all.
           0
         elsif (d = result.first[:description]).nil?
-          # There's a table, it was just created before the migration system
-          # existed.
-          1
+          # The table exists but the version comment is missing
+          if _db_schema_matches_db_version_1?
+            1
+          else
+            _raise_db_version_comment_missing_error
+          end
```

Comment:
Is there really a use case for continuing to support the version 1 schema? It’s going to be fundamentally incompatible with the rest of the code base at this point. Surely we can just raise if the comment is missing?

Reply:
Yeah, I guess not, actually. The migrations system was introduced in [v0.5.0](https://github.com/que-rb/que/blob/master/CHANGELOG.md#050-2014-01-14) which came out on 2014-01-14. With the latest version of 0.x being v0.14.3 (2018-03-02), I guess we can assume no one's on a version below v0.5.0.

I'll change the PR to assume DB schema version 1 would have the comment, and just add a note in the changelog that support for upgrading directly from versions below v0.5.0 is unsupported.